src/ankh/cmd/ankh/main.go

```diff
@@ -261,3 +272,43 @@ func check(err error) {
 		os.Exit(1)
 	}
 }
+
+func runScripts(ctx *ankh.ExecutionContext, scripts []struct { Path string }) {
+	for _, script := range scripts {
+		path := script.Path
+		if path == "" {
+			log.Warnf("Missing path in script %s", script)
+			break
+		}
+		log.Infof("Running script: %s", path)
+		if ctx.DryRun == true {
+			log.Infof("- OK (dry) %s", path)
+			break
+		}
+		// pass kube context and the "global" config as a yaml environment variable
+		// also send the kube context as the first argument to the script
+		cmd := exec.Command(path, ctx.AnkhConfig.CurrentContext.KubeContext)
```

Comment:
let's rely solely on env vars, and omit context as an argument. if we have use cases depending on the arg, we can convert them to use the env var.

Reply:
Yeah I was going to bring this up too, 👍 for just env vars